lib/tool/src/hand_tool/hand_tool_provider.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:paintroid/command/src/command_factory_provider.dart';
+import 'package:paintroid/command/src/command_manager_provider.dart';
+import 'package:paintroid/tool/src/brush_tool/brush_tool_state_provider.dart';
+import 'package:paintroid/tool/src/hand_tool/hand_tool.dart';
+import 'package:paintroid/tool/src/tool_types.dart';
+import 'package:riverpod_annotation/riverpod_annotation.dart';
+
+part 'hand_tool_provider.g.dart';
+
+@riverpod
+HandTool handTool(HandToolRef ref) {
+  return HandTool(
+    paint: ref.watch(brushToolStateProvider.select((state) => state.paint)),
+    type: ToolType.HAND,
```

Comment:
Is there a need to pass the type via the constructor? As the type does not change during the tools lifetime. Consider using a const/final instance variable. 

p.s.: also adjust other tools

Reply:
Please create the ticket and send it to me

Reply:
👍🏻